addons/io_scene_gltf2/blender/exp/gltf2_blender_gather_primitive_attributes.py

```diff
@@ -165,10 +165,10 @@ def __gather_skins(blender_primitive, export_settings):
         joint_id = 'JOINTS_' + str(bone_index)
         weight_id = 'WEIGHTS_' + str(bone_index)
         while blender_primitive["attributes"].get(joint_id) and blender_primitive["attributes"].get(weight_id):
-            if bone_index >= 4:
-                gltf2_io_debug.print_console("WARNING", "There are more than 4 joint vertex influences."
-                                                        "Consider to apply blenders Limit Total function.")
+            if bone_index >= 1:
```

Comment:
Why `>= 1` here?

Reply:
Heh, was pretty confused about why the 4 wasn't working, however bone_index here refers to the output groups in gltf (JOINTS_X, WEIGHTS_X) which each hold 4 bones.

Reply:
Oh that's confusing. 😅 Do you mind renaming this variable to `group_index` or `set_index`?